modules/ztp-install-factory-spoke-pipeline.adoc

```diff
@@ -0,0 +1,186 @@
+        bmc_pass: "user-pass"
+        root_disk: sda
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+    worker0: <13>
+        nic_ext_dhcp: eno4
+        nic_int_static: eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:19"
+        mac_int_static: "aa:ss:dd:ee:b1:19"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+  - spoke2-name:
+      master0:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:20"
+        mac_int_static: "aa:ss:dd:ee:b1:20"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master1:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:21"
+        mac_int_static: "aa:ss:dd:ee:b1:21"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master2:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:22"
+        mac_int_static: "aa:ss:dd:ee:b1:22"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      worker0:
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:29"
+        mac_int_static: "aa:ss:dd:ee:b1:29"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+----
+<1> This option is configurable and sets the name of the edge cluster.
+<2> This value must match `master0`, `master1` or `master2`.
+<3> Optional: Interfaces to ignore in the host.
+<4> NIC connected to the external DHCP.
+<5> NIC interface name connected to the internal network.
+<6> MAC address for the NIC connected to the external DHCP network.
+<7> MAC address for the NIC connected to the internal static network
+<8> URL for the Baseboard Management Controller (BMC).
+<9> The BMC username.
+<10> The BMC password.
+<11> Mandatory: Disk device to be used for operating system installation.
+<12> List of disk available in the node to be used for storage.
+<13> Hardcoded name set as `worker0` for the worker node.
+
+. Set the following environment variable:
++
+[source,terminal]
+----
+$ export KUBECONFIG=<path_to_kubeconfig>/kubeconfig-file
+----
+
+. Start the edge cluster pipeline from the command line:
++
+[source,terminal]
+----
+$ tkn pipeline start -n spoke-deployer -p git-revision=tekton spokes-config="$(cat /path-to-spoke-yaml/spokes.yaml)" -p kubeconfig=${KUBECONFIG} -w=ztp,claimName=ztp-pvc --timeout 5h --use-param-defaults deploy-ztp-spokes
```

Comment:
we no longer specify the branches with -p . You can remove the "-p git-revision=tekton"